### PR TITLE
git checkouts 3 - tcpdump, swaks, strace, squid, skalibs

### PR DESCRIPTION
**Pull request by smoser**
move to git checkout

### CODE REVIEW

File: squid.yaml [hunk 1]
```diff
@@ -15,19 +15,30 @@ environment:
       - build-base
       - busybox
       - ca-certificates-bundle
+      - ed # busybox ed is not sufficient for boostrap.sh use
```
Comment (dakaneye) [inline squid.yaml:18]: incomplete comment?

**Review by dakaneye — DISMISSED** [review]
LGTM

File: squid.yaml (hunk 1 above)
Comment (smoser) [reply squid.yaml]: updated.

**Review by krishjainx — APPROVED** [review]
LGTM as well!

**Pull request merged**